docker-compose.yaml

```diff
@@ -3,24 +3,47 @@ services:
   mongodb:
     image: 'mongo:3.4.9'
     container_name: mongodb
+    restart: always
     ports:
       - '27017:27017'
     volumes:
       - data:/data/db
+      - mongologs:/var/log/mongodb
     environment:
-      MONGO_INITDB_DATABASE: VisitorManagement
-      MONGO_INITDB_ROOT_USERNAME: admin
-      MONGO_INITDB_ROOT_PASSWORD: password
+      MONGO_INITDB_DATABASE: db_zvisitor
+      MONGO_INITDB_ROOT_USERNAME: $MONGO_INITDB_ROOT_USERNAME
+      MONGO_INITDB_ROOT_PASSWORD: $MONGO_INITDB_ROOT_PASSWORD
+    command: ['--logpath', '/var/log/mongodb/mongo_logs.log']
   backend:
     build: .
     container_name: backend
+    restart: always
     ports:
       - '8080:8080'
+    volumes:
+      - logs:/logs/zv
     environment:
-      MONGODB_USERNAME: admin
-      MONGODB_PASSWORD: password
+      FILE_UPLOAD: /home/akshay.h@ah.zymrinc.com/Desktop/zvisitor_data
+      MONGODB_DATABASE: db_zvisitor
+      MONGODB_HOST: mongodb
+      AUTHENTICATION_DATABASE: admin
+      VALID_MAIL: '@zymr.com'
+      ORG_DEPARTMENT_EMAIL: zymrtest@gmail.com
+      USER_APP_PORT: 8080
+      MAIL_PERSONAL: z-visitor
+      ADMIN_MAIL: $ADMIN_MAIL
+      ADMIN_PASSWORD: $ADMIN_PASSWORD
+      SLACK_TOKEN: $SLACK_TOKEN
+      SLACK_USERNAME: $SLACK_USERNAME
+      MAIL_USERNAM: $MAIL_USERNAM
+      MAIL_PASSWORD: $MAIL_PASSWORD
+      MONGODB_USERNAME: $MONGODB_USERNAME
+      MONGODB_PASSWORD: $MONGODB_PASSWORD
```

Comment:
Good

Reply:
@Akshaykumar-Hingarajiya  As we use the name volume so add volume path details in the ReadMe file.

Reply:
Done.